<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="groupBoxConnexion">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>711</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string>Connexion</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Adresse IP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>86</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom de la bdd</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>33</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mot de passe</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditIP">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>172.18.58.14</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNomBdd">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>banque</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditLogin">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>snir</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditMDP">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>170</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>snir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonConnexionBDD">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>50</y>
      <width>190</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Connexion à la base de donnée</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxAffichageInfo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>711</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>Affichage infos comptes</string>
   </property>
   <widget class="QPushButton" name="pushButtonAfficherInfo">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>274</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Afficher les informations de tout les comptes</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidgetInfos">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>641</width>
      <height>131</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Nom</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Prénom</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Numéro de compte</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Solde</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonQuitter">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>570</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonQuitter</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>528</y>
    </hint>
    <hint type="destinationlabel">
     <x>517</x>
     <y>575</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
